go.mod

```diff
@@ -27,4 +27,7 @@ require (
 	nhooyr.io/websocket v1.7.4
 )
 
+// Temporary workaround to support protected (json-gold issue 22)
+replace github.com/piprate/json-gold v0.2.0 => github.com/trustbloc/json-gold v0.2.1-0.20191211165818-ef1958eae4be
```

Comment:
I assume this branch should work: https://github.com/piprate/json-gold/tree/json11-dec-2019

Reply:
Agree, I will change this!